web/ui/static/css/targets.css

```diff
@@ -1,11 +1,52 @@
-tr.job_header {
+h2.job_header {
     font-size: 20px;
-    padding-top: 10px;
     font-weight: bold;
-    padding-bottom: 10px;
     cursor: pointer;
 }
 
-tr.job_details > td{
-    padding: 0 !important;
-}
\ No newline at end of file
+h2.danger a {
+    color: rgb(242, 65, 65);
+}
+
+.filters button.unhealthy-targets {
```

Comment:
This checkbox button business should probably be refactored into a shared CSS file, but I think that doesn't need to happen in this PR.

Reply:
I <3 refactoring, so happy to do that after this is done!

Reply:
Cool, same applies to the related JS :)